app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -306,7 +303,11 @@ class BrowserTabViewModel(
 
     private val autoCompletePublishSubject = PublishRelay.create<String>()
     private val fireproofWebsiteState: LiveData<List<FireproofWebsiteEntity>> = fireproofWebsiteRepository.getFireproofWebsites()
+
+    @ExperimentalCoroutinesApi
+    @FlowPreview
     private val showPulseAnimation: LiveData<Boolean> = ctaViewModel.showFireButtonPulseAnimation.asLiveData(context = viewModelScope.coroutineContext)
```

Comment:
Nit: I'm getting a lint warning here because the line is too long (by 1 character).

Reply:
🤔 I was using hard wrap at 160 cols...I could have been using the wrong config.

Reply:
Not at all actually. That changed in https://github.com/duckduckgo/Android/pull/761 from 150 to 160. Not sure if it was deliberate or a mistake. I'll create a task to discuss, feel free to go back to 160 in the meantime.